lib/shared/constants.dart

```diff
@@ -0,0 +1,3 @@
+class ConstantsClass{
+  static const String appFont='Handjet', textFont='Tektur',appName='The MovieDB App';
```

Comment:
Separate const in different lines, and add EOF

Reply:
solved